source/gui/keybinding_manager.py

```
@@ -0,0 +1,372 @@
+        if new_state:
+            self.config["--delay"]["active"] = False
+            self.config["--set"]["active"] = False
+            self.log("[KEYBIND ACTION] Random mode: ON")
+        else:
+            self.log("[KEYBIND ACTION] Random mode: OFF")
+        
+        # Update UI if flags panel is available
+        try:
+            if hasattr(self.event_handlers.ui.get('flags_panel'), 'random_mode'):
+                self.event_handlers.ui['flags_panel'].random_mode.set(new_state)
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)